### PR TITLE
Allow language selection

### PR DESCRIPTION
**Pull request by MattIPv4**
## Type of Change

- **Tool Source:** Vue, JS & SCSS
- **Something else:** Deps

## What issue does this relate to?

cc #190 

### What should this PR do?

With #190 we're adding strings for a second language, Chinese, to the tool. This PR adds a dropdown in the header to be able to select a language and updates the logic throughout the tool to make use of vue-i18n and the translation strings properly.

### What are the acceptance criteria?

- All strings throughout the tool load correctly in English
- Changing the tool language updates all strings correctly
- Changing the tool language is saved to the URL
- Loading the tool with a valid language set in the URL works
- Loading the tool with an invalid language set in the URL defaults to English

### CODE REVIEW

File: src/nginxconfig/i18n/zh-cn/templates/app.js [hunk 1]
```diff
@@ -29,6 +29,9 @@ import common from '../common';
 export default {
     title: `${common.nginx} 配置`,
     description: `配置高性能、安全、稳定的${common.nginx}服务器的最简单方法。`,
+    en: '英语', // TODO: translate
+    zhCN: '简体中文', // TODO: translate
+    zhTW: '繁体中文', // TODO: translate
```
Comment (MattIPv4) [inline src/nginxconfig/i18n/zh-cn/templates/app.js:34]: 👋 @moniang I was wondering if you could check these translations are correct (or suggest new ones)?

The source strings are:
```
    en: 'English',
    zhCN: 'Chinese (simplified)',
    zhTW: 'Chinese (traditional)',
```

File: src/nginxconfig/i18n/zh-tw/templates/app.js [hunk 2]
```diff
@@ -29,6 +29,9 @@ import common from '../common';
 export default {
     title: `${common.nginx} 配寘`,
     description: `配寘高性能、安全、穩定的${common.nginx}服務器的最簡單方法。`,
+    en: '英語', // TODO: translate
+    zhCN: '簡體中文', // TODO: translate
+    zhTW: '繁體中文', // TODO: translate
```
Comment (MattIPv4) [inline src/nginxconfig/i18n/zh-tw/templates/app.js:34]: 👋 @moniang I was wondering if you could check these translations are correct (or suggest new ones)?

The source strings are:
```
    en: 'English',
    zhCN: 'Chinese (simplified)',
    zhTW: 'Chinese (traditional)',
```

Comment (moniang) [reply src/nginxconfig/i18n/zh-tw/templates/app.js:34]: yes,no problem

File: src/nginxconfig/i18n/zh-cn/templates/app.js (hunk 1 above)
Comment (moniang) [reply src/nginxconfig/i18n/zh-cn/templates/app.js:34]: Yes, no problem

File: src/nginxconfig/i18n/zh-tw/templates/app.js (hunk 2 above)
Comment (moniang) [reply src/nginxconfig/i18n/zh-tw/templates/app.js:34]: Yes, no problem

**Review by JakeMakesStuff — APPROVED** [review]
LGTM.

**Pull request merged**